helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -32,7 +32,7 @@ spec:
       annotations:
         description: 'Prometheus is not sending data to Grafana Cloud.'
         opsrecipe: tbd/
-      expr: absent(prometheus_remote_storage_samples_total{remote_name="grafana-cloud"})
+      expr: absent(prometheus_remote_storage_samples_total{remote_name="grafana-cloud", cluster_type="management_cluster"})
```

Comment:
I think we wanted to add the pipeline and installation and maybe cluster-id from values ?